pom.xml

```diff
@@ -268,17 +268,32 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
```

Comment:
We don't need the ua parser dependency for AS anymore, user-agent parsing is done in DAS